test/unit/math/rev/functor/ode_test_util.hpp

```diff
@@ -0,0 +1,451 @@
+#ifndef STAN_MATH_TEST_ODE_TEST_UTIL_HPP
+#define STAN_MATH_TEST_ODE_TEST_UTIL_HPP
+
+#include <stan/math/rev/core.hpp>
+#include <stan/math/prim.hpp>
+#include <test/unit/util.hpp>
+#include <gtest/gtest.h>
+#include <limits>
+#include <sstream>
+#include <vector>
+
+// calculates finite diffs for ode solver with varying parameters
```

Comment:
Is there any chance of using this to replace `test/unit/math/rev/functor/util_rk45.hpp` and `test/unit/math/rev/functor/util_cvodes_bdf.hpp` in this pull?

Reply:
yes that's why I did it this way. But IMO it should be part of a new PR dedicated to refactor ODE tests.